test/burials/validation.unit.spec.js

```diff
@@ -37,12 +37,38 @@ describe('Burials validation', () => {
           addError: sinon.spy()
         }
       };
-      validateBurialDate(errors, {
+      validateBurialAndDeathDates(errors, {
         deathDate: '2017-01-02',
         burialDate: '2017-01-01'
       });
 
       expect(errors.burialDate.addError.called).to.be.true;
     });
+    it('should allow birth date before death date', () => {
+      const errors = {
+        deathDate: {
+          addError: sinon.spy()
+        }
+      };
+      validateBurialAndDeathDates(errors, {
+        deathDate: '2017-01-02',
+        veteranDateOfBirth: '2017-01-01'
+      });
+
+      expect(errors.deathDate.addError.called).to.be.false;
+    });
+    it('should not allow death date after birth date', () => {
```

Comment:
Methinks this description is backwards. I think you meant `should not allow death date before birth date`.